src/software/proto/message_translation/ssl_referee.h

```diff
@@ -24,3 +25,12 @@ RefboxGameState createRefboxGameState(const Referee &packet, TeamColour team_col
  * @return RefboxStage from stage
  */
 RefboxStage createRefboxStage(const Referee &packet);
+
+/**
+ * Extracts the ball placement point from Referee packet
+ *
+ * @param packet Referee protobuf
+ *
+ * @return ball placement point if found
+ */
+std::optional<Point> createBallPlacementPoint(const Referee &packet);
```

Comment:
Should this be named `getBallPlacementPoint` since it's extracting / getting the data?

Reply:
fixed